src/main/java/org/waarp/openr66/context/authentication/R66Auth.java

```diff
@@ -93,16 +93,28 @@ protected NextCommandReply setBusinessPassword(String arg0)
      * @throws Reply421Exception
      *             If the service is not available
      */
-    public boolean connection(DbSession dbSession, String hostId, byte[] arg0)
+    public boolean connection(DbSession dbSession, String hostId, byte[] arg0,
```

Comment:
One of the goal of the new Database management is to hide the database use behind DAOs.
DbSession is kept but tagged a Deprecated to allow for a smooth evolution of the source code.
And to avoid to impact all of the dependent projects.

Reply:
OK so I would propose:
- add a signature without DbSession with this code
- call the new signature one from the old one, so ignoring the DbSession and adding Deprecated to it
Then we could trace easier the change to be done elsewhere (also valid for others DAO of course).

Reply:
You are right I will add this.

Reply:
OK Thanks !